source/geometry_model/initial_topography_model/ascii_data.cc

```diff
@@ -78,19 +78,16 @@ namespace aspect
         {
           // No need to set the radial coordinate correctly,
           // because it will be thrown away in get_data_component anyway
-          std::array<double, dim> point;
-          point[0] = 6371000.0;
+          global_point[0] = 6371000.0;
```

Comment:
This is pre-existing, but we should probably query the geometry model for the proper radius. Though in the comment above, it says that we don't really care, so we may set it to zero.

Reply:
zero would be a bad idea for a radius, as then phi and theta would not be defined. I was considering a value of 1, but actually we don't need to change this text now.